src/components/styles/_key.less

```diff
@@ -1,87 +1,14 @@
-.key-btn.pink-btn {
-  border-top: 2px solid @pink;
-  background-color: @pink;
+.btnVariant(@color, @keyHoverColor, @keyBoxColor) {
+  border-top: 2px solid @color;
+  background-color: @color;
   box-shadow: inset 0px -12px @keyBoxPink;
```

Comment:
You are using `@keyBoxPink` in the mixin instead of `@keyBoxColor` please fix it.